test/e2e/generate_kube_test.go

```diff
@@ -555,6 +556,15 @@ var _ = Describe("Podman generate kube", func() {
 		kube.WaitWithDefaultTimeout()
 		Expect(kube).Should(Exit(0))
 
+		b, err := os.ReadFile(outputFile)
+		Expect(err).To(BeNil())
```

Comment:
```suggestion
		Expect(err).ShouldNot(HaveOccurred())
```
Produces a more meaningful error message.